src/messaging/CHIPReliableMessageContext.cpp

```diff
@@ -0,0 +1,454 @@
+/*
+ *    Copyright (c) 2020 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+/**
+ *    @file
+ *      This file implements the CHIP reliable message protocol.
+ *
+ */
+
+#include <inttypes.h>
+
+#include <core/CHIPEncoding.h>
+#include <messaging/CHIPMessageLayerPrivate.h>
+#include <messaging/CHIPMessageLayerUtils.h>
+#include <messaging/CHIPReliableMessageContext.h>
```

Comment:
please place the module header as the first include

Reply:
done